classes/submission/reviewAssignment/Collector.php

```diff
@@ -435,8 +445,24 @@ public function getQueryBuilder(): Builder
 
         $q->when(
             $this->reviewerIds !== null,
-            fn (Builder $q) =>
-            $q->whereIn('ra.reviewer_id', $this->reviewerIds)
+            fn (Builder $q) => $q
+                ->whereIn('ra.reviewer_id', $this->reviewerIds)
+                ->when(
+                    $this->isLastReviewRoundByReviewer,
+                    fn (Builder $q) => $q
+                        // Aggregate the latest review round number for the given assignment and reviewer
+                        ->leftJoinSub(
+                            DB::table('review_assignments as ramax')
+                                ->leftJoin('review_rounds as rrmax', 'ramax.review_round_id', '=', 'rrmax.review_round_id')
```

Comment:
Interestingly review assignment seems to have both roundId and round columns, so probably no need to join it with review rounds?

Reply:
Yes, you are right